src/Microsoft.AspNet.Mvc.WebApiCompatShim/Formatters/HttpResponseMessageOutputFormatter.cs

```diff
@@ -42,6 +42,10 @@ public bool CanWriteResult(OutputFormatterContext context, MediaTypeHeaderValue
 
             using (responseMessage)
             {
+                // Ignore the chunked Transfer-Encoding header if its set by the user. 
+                // We let the host decide about whether the response should be chunked or not.
+                responseMessage.Headers.TransferEncodingChunked = false;
```

Comment:
Do we need a check here? There are other transfer-encoding values. Take a look at the old WebAPI code for inspiration.